client/app/views/templates/view_register_preset.jade

```diff
@@ -24,11 +24,19 @@ p#preset-email-tip.tips
 
 
 label.with-input(for="preset-timezone", aria-describedby="preset-timezone-tip")
-    span= t('preset timezone')
-    input#preset-timezone(type="text", name="timezone", list="preset-timezone-datalist", required=true)
-    datalist#preset-timezone-datalist
-        each timezone in timezones
-            option(value=timezone)
+    select#preset-timezone(name="timezone" required=true)
+        - var skipstep  = false
+        - var lastcont = null
+        each timezone, index in timezones
+            unless skipstep
+                - var cont = timezone.split('/')[0]
+                if cont != lastcont
+                    - lastcont = cont
+                    if index
+                        option(disabled=true)= '---'
+                if cont.toLowerCase() == 'africa'
```

Comment:
Its starts by a "A" :smile:
More seriously, we first presents 4 Europe's timezones, then 2 America's ones, and finally display all available timezones in alphabetical order. This trick is just to separate the 2 first blocks from the rest of the list.